Sources/APNSwift/APNSPushType.swift

```diff
@@ -91,4 +92,6 @@ public struct APNSPushType: Hashable {
     ///
     /// - Important: If you set this push type, you must use the topic from the UID attribute in the subject of your MDM push certificate.
     public static let mdm = Self(configuration: .mdm)
+    
+    public static let liveactivity = Self(configuration: .liveactivity)
```

Comment:
Taking from the Apple docs
```suggestion
    /// Use the live activity push type to update your live activity.
    ///
    /// - Important: If you set this push type, the topic must use your app’s bundle ID with `.push-type.liveactivity` appended to the end.
    public static let liveactivity = Self(configuration: .liveactivity)
```